src/ZendDiagnostics/Check/Redis.php

```diff
@@ -55,16 +62,26 @@ private function createClient()
     {
         if (class_exists('\Redis')) {
             $client = new RedisExtensionClient();
-            $client->connect($this->host);
+            $client->connect($this->host, $this->port);
+
+            if ($this->auth) {
+                $client->auth($this->auth);
```

Comment:
Please add [return value check](https://github.com/phpredis/phpredis#auth). If the authentication fails, IMO the test should return a Failure.